projects/plugins/jetpack/modules/custom-css/custom-css/preprocessors/lessc.inc.php

```diff
@@ -3009,6 +3022,9 @@ protected function color(&$out) {
 	// arguments are separated by , unless a ; is in the list, then ; is the
 	// delimiter.
 	protected function argumentDef(&$args, &$isVararg) {
+		$value  = null;
+		$rhs    = null;
+		$newArg = null;
```

Comment:
If this is doing anything at all, it would seem to be masking a logic error. Better would be to place it somewhere around line 3071, as if it's really possible to miss all the assignments down there then it's probably not correct to be reusing whatever happened to be set from an earlier pass through the loop.

Reply:
$value and $rhs are output parameters that rector wants to be "initialized" before they're used on 3041 and 3046; 
$newArg could be initialized on each loop, but that might change the way the program works. I don't think there's anything wrong with $newArg in the way the program works, rector simply doesn't like it when path A doesn't set $newArg, path B sets $newArg, and both can land on an `if ( $newArg )`

Reply:
Yeah, the `$newArg` bit might change how it works, although if it does I'd be surprised it works correctly now. But I'll go along with "better to not risk it".